src/components/helper/validation.helper.js

```diff
@@ -1,33 +1,80 @@
-const { ValidationError } = require('../../errors');
+const _ = require('lodash');
```

Comment:
I do not like to use more lodash, this is one of the packages that should remove. We have spent time to replace lodash stuff with es6 methodes, in past.

I am a litte bit confused over the location and naming. We have utils (not helper) that we can use global. Yes we have some miss placed global helper/ but they should move to utils too. 
But in component it should not exist a global component utils folder. 
This invalidate the sense of the component folder.

Reply:
Added SC-164